sandboxneu/scheduleneu
# fix-parser

Moved logic for different parts of the degree audit into individual functions.
Fixed the logic for producing the courses that are required.

To test:
Run the code as per the README of this repository.
Ensure that the degree audit produced is as desired.
